src/view/event-edit-view.js

```diff
@@ -0,0 +1,134 @@
+import {createElement} from '../render.js';
```

Comment:
```suggestion
import { createElement } from '../render.js';
```

Старайся писать однообразно)